
rotate_kP: 4.0
rotate_kI: 0.0
rotate_kD: 0.0

max_angular_vel: 10.0
max_linear_vel: 4.0
max_linear_accel: 2.0

linear_kP: 1.0
linear_kI: 0.0
linear_kD: 0.0

object_kF: 0.9

camera_base_frame: camera_link

object_reached_threshold: 0.2
command_rate: 15.0
no_object_timeout: 5.0
odom_frame: odom
base_frame: base_link

enable_linear_vel: true
enable_prediction: false


laser_obstacle_lower_angle: -1.5707
laser_obstacle_upper_angle: 1.5707
laser_obstacle_threshold: 2.0
